contracts/multisig/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './artifacts';
```

Comment:
The lack of an __index.ts__ file in the `@0x/contracts-multisig` package was the other culprit in causing `test-publish` to fail. The __package.json__ file in the `@0x/contracts-multisig` specifies `lib/src/index.js` as the entry point, but that file did not exist, so there was an error when you try to import the `@0x/contracts-multisig` package.